distribution/lib/Standard/Table/0.2.32-SNAPSHOT/src/Data/Column_Selector.enso

```diff
@@ -0,0 +1,29 @@
+from Standard.Base import all
+
+from Standard.Table.Data.Matching import all
```

Comment:
Technically only need to import `Matching_Strategy` - as a type definition probably worth being tight.

Reply:
Well, we are also using `Exact` imported from there. Then I'd need to change it to `Matching.Exact`.

What we import is not exported by default (need an explicit `export` statement), so I don't think it matters really. But can change if you think that's better.

Reply:
We should use "all" imports for prelude like stuff only. This was described in the Language document by my and Ara some time ago. The "all" import should be rare as it pollutes the scope a lot.